index.html

```diff
@@ -1,15 +1,32 @@
 <!DOCTYPE html>
-<html>
+<html data-ng-app="wdiRadio">
   <head>
     <title>WDI Radio</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.15/angular-ui-router.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.0-beta.2/angular-resource.min.js"></script>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-cookie/4.0.9/angular-cookie.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/ng-token-auth/0.0.28/ng-token-auth.min.js"></script>
+
+
     <script src="js/app.js"></script>
+    <script src="js/songs/songs.js"></script>
+    <script src="js/songs/index.controller.js"></script>
+    <script src="js/songs/song.factory.js"></script>
+    <script src="js/songs/show.controller.js"></script>
+    <script src="js/songs/form.directive.js"></script>
+    <script src="js/songs/song.directive.js"></script>
```

Comment:
what is the purpose of this song directive, might be better to name it like `player` or `show`